README.md

```diff
@@ -110,20 +110,11 @@ Also remove some shaders mods:
 
 The `distutils` module is required to install Python packages but it was removed in Python 3.12. You can still use it by installing `setuptools` (inside the venv): `pip install setuptools`
 
-### rarfile.RarCannotExec: Cannot find working tool
```

Comment:
This is still possible on previous version of the tool and should not be removed